### PR TITLE
add micro-transaction example

### PR DESCRIPTION
**Pull request by pvdrz**

### CODE REVIEW

File: examples/outputs/microtransaction.rs [hunk 1]
```diff
@@ -0,0 +1,79 @@
+// Copyright 2022 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+//! cargo run --example foo --release
```
Comment (Thoralf-M) [inline examples/outputs/microtransaction.rs]: Wrong name

File: examples/outputs/microtransaction.rs [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+            },
+            BasicOutputBuilder,
+        },
+        payload::milestone::MilestoneIndex,
+    },
+    secret::{mnemonic::MnemonicSecretManager, SecretManager},
+    Client, Result,
+};
+
+/// In this example we will send basic outputs with different feature blocks
```
Comment (Thoralf-M) [inline examples/outputs/microtransaction.rs]: Maybe say what's the purpose of them?

File: examples/outputs/microtransaction.rs [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+
+const NODE_URL: &'static str = "http://localhost:14265";
```
Comment (Thoralf-M) [inline examples/outputs/microtransaction.rs:32]: Wen also load from .env? :trollface:

Comment (pvdrz) [reply examples/outputs/microtransaction.rs:32]: Soon™

Edit: upcoming PR

**Review by Thoralf-M — APPROVED** [review]

**Review by thibault-martinez — APPROVED** [review]

**Pull request merged**